### PR TITLE
Changelog for v1.11.2

### PR DESCRIPTION
**Pull request by zeripath**
Create the changelog for v1.11.2

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,51 @@ This changelog goes through all the changes that have been made in each release
...
 been added to each release, please refer to the [blog](https://blog.gitea.io).
 
+## [1.11.2](https://github.com/go-gitea/gitea/releases/tag/v1.11.2) - 2020-03-06
+
+* BREAKING
+  * Various fixes in login sources (#10428) (#10429)
+* SECURITY
+  * Ensure only own addresses are updated (#10397) (#10399)
+* BUGFIXES
+  * Fix modifying issues (#10623) (#10626)
```
Comment (guillep2k) [inline CHANGELOG.md]:
```suggestion
  * Allow users with write permissions to modify issue descriptions and comments. (#10623) (#10626)
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -4,6 +4,51 @@ This changelog goes through all the changes that have been made in each release
...
+  * Handle deleted base branch in PR (#10618) (#10619)
+  * Delete dependencies when deleting a repository (#10608) (#10616)
+  * Ensure executable bit is kept on the web editor (#10607) (#10614)
+  * Update mergebase in pr checker (#10586) (#10605)
+  * Logout POST action (#10582) (#10585)
+  * Fix migration bug on v96.go (#10572) (#10573)
```
Comment (guillep2k) [inline CHANGELOG.md]: Maybe we should be more explicit here?
```suggestion
  * Fix release attachments being deleted while upgrading (#10572) (#10573)
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -4,6 +4,51 @@ This changelog goes through all the changes that have been made in each release
...
+  * Bugfix/slack webhook redirect backport (#10566)
```
Comment (guillep2k) [inline CHANGELOG.md]:
```suggestion
  * Fix redirection path if Slack webhook channel is invalid (#10566)
```

File: CHANGELOG.md [hunk 4]
```diff
@@ -4,6 +4,51 @@ This changelog goes through all the changes that have been made in each release
...
+  * head.tmpl og:image picture location (#10531) (#10556)
+  * Fix 404 after activating secondary email (backport of #10547) (#10553)
```
Comment (guillep2k) [inline CHANGELOG.md]:
```suggestion
  * Fix 404 after activating secondary email (#10547) (#10553)
```

File: CHANGELOG.md [hunk 5]
```diff
@@ -4,6 +4,51 @@ This changelog goes through all the changes that have been made in each release
...
+  * Show Signer in commit lists and add basic trust (#10425 & #10511) (#10524)
+  * Fix potential bugs (#10513) (#10518)
+  * Org action fixes and form cleanup (#10512) (#10514)
+  * Use \[:space:\] instead of \\s (#10508) (#10509)
+  * Avoid mailing explicit unwatched (#10475) (#10500)
```
Comment (guillep2k) [inline CHANGELOG.md]:
```suggestion
  * Avoid mailing users that have explicitly unwatched an issue (#10475) (#10500)
```

File: CHANGELOG.md [hunk 6]
```diff
@@ -4,6 +4,51 @@ This changelog goes through all the changes that have been made in each release
...
+  * Handle push rejection message in Merge & Web Editor (#10373) (#10497)
+  * Fix SQLite concurrency problems by using BEGIN IMMEDIATE (#10368) (#10493)
+  * Fix double PR notification from API (#10482) (#10486)
+  * Fix admin notices (#10480) (#10483)
+  * Change admin dashboard to POST (#10465) (#10466)
+  * Change action GETs to POST (#10462) (#10464)
+  * Show the username as a fallback on feeds if full name is blank (#10461)
+  * Update markbates/goth (backport v1.11) (#10445)
+  * Trigger webhooks on issue label-change via API too (#10421) (#10439)
+  * Fix webhook bug (#10427) (#10432)
```
Comment (guillep2k) [inline CHANGELOG.md]:
```suggestion
  * Fix git reference type in webhooks (#10427) (#10432)
```

File: CHANGELOG.md [hunk 7]
```diff
@@ -4,6 +4,51 @@ This changelog goes through all the changes that have been made in each release
...
+  * Prevent panic on merge to PR (#10403) (#10408)
+  * Fix wrong num closed issues on repository when close issue via commit… (#10364) (#10380)
+  * Reading pull attachments should depend on read UnitTypePullRequests (#10346) (#10354)
+  * Set max-width on review-box comment box (#10348) (#10353)
+  * Prevent nil pointer in GetPullRequestCommitStatusState (#10342) (#10344)
+  * Fix status check enable (#10341) (#10343)
```
Comment (guillep2k) [inline CHANGELOG.md]:
```suggestion
  * Fix protected branch status check settings (#10341) (#10343)
```

File: CHANGELOG.md [hunk 8]
```diff
@@ -4,6 +4,51 @@ This changelog goes through all the changes that have been made in each release
...
+  * Fix potential bugs (#10513) (#10518)
+  * Org action fixes and form cleanup (#10512) (#10514)
+  * Use \[:space:\] instead of \\s (#10508) (#10509)
+  * Avoid mailing explicit unwatched (#10475) (#10500)
+  * Handle push rejection message in Merge & Web Editor (#10373) (#10497)
+  * Fix SQLite concurrency problems by using BEGIN IMMEDIATE (#10368) (#10493)
+  * Fix double PR notification from API (#10482) (#10486)
+  * Fix admin notices (#10480) (#10483)
+  * Change admin dashboard to POST (#10465) (#10466)
+  * Change action GETs to POST (#10462) (#10464)
```
Comment (jolheiser) [inline CHANGELOG.md]: Security

File: CHANGELOG.md [hunk 9]
```diff
@@ -4,6 +4,51 @@ This changelog goes through all the changes that have been made in each release
...
+  * Ensure executable bit is kept on the web editor (#10607) (#10614)
+  * Update mergebase in pr checker (#10586) (#10605)
+  * Logout POST action (#10582) (#10585)
+  * Fix migration bug on v96.go (#10572) (#10573)
+  * Bugfix/slack webhook redirect backport (#10566)
+  * head.tmpl og:image picture location (#10531) (#10556)
+  * Fix 404 after activating secondary email (backport of #10547) (#10553)
+  * Show Signer in commit lists and add basic trust (#10425 & #10511) (#10524)
+  * Fix potential bugs (#10513) (#10518)
+  * Org action fixes and form cleanup (#10512) (#10514)
```
Comment (jolheiser) [inline CHANGELOG.md]: Security

File: CHANGELOG.md [hunk 10]
```diff
@@ -4,6 +4,51 @@ This changelog goes through all the changes that have been made in each release
...
+  * Various fixes in login sources (#10428) (#10429)
+* SECURITY
+  * Ensure only own addresses are updated (#10397) (#10399)
+* BUGFIXES
+  * Fix modifying issues (#10623) (#10626)
+  * Handle deleted base branch in PR (#10618) (#10619)
+  * Delete dependencies when deleting a repository (#10608) (#10616)
+  * Ensure executable bit is kept on the web editor (#10607) (#10614)
+  * Update mergebase in pr checker (#10586) (#10605)
+  * Logout POST action (#10582) (#10585)
```
Comment (jolheiser) [inline CHANGELOG.md]: Security

**Review by jolheiser — CHANGES_REQUESTED** [review]
Please update https://github.com/go-gitea/gitea/blob/80db44267ccb688c596e8375523af5cd92864d87/docs/config.yaml#L21 to `1.11.2` as well. :slightly_smiling_face: 

For ones I commented "Security" they can be merged into one if you'd like since they are all the same concept of fix.

File: CHANGELOG.md [hunk 11]
```diff
@@ -4,6 +4,51 @@ This changelog goes through all the changes that have been made in each release
...
+  * Fix 404 after activating secondary email (#10547) (#10553)
+  * Show Signer in commit lists and add basic trust (#10425 & #10511) (#10524)
+  * Fix potential bugs (#10513) (#10518)
+  * Use \[:space:\] instead of \\s (#10508) (#10509)
+  * Avoid mailing users that have explicitly unwatched an issue (#10475) (#10500)
+  * Handle push rejection message in Merge & Web Editor (#10373) (#10497)
+  * Fix SQLite concurrency problems by using BEGIN IMMEDIATE (#10368) (#10493)
+  * Fix double PR notification from API (#10482) (#10486)
+  * Fix admin notices (#10480) (#10483)
+  * Change admin dashboard to POST (#10465) (#10466)
```
Comment (jolheiser) [inline CHANGELOG.md]: These two are also security. Sorry, multi-line comments are a bit weird and not very apparent. :sweat_smile:

File: CHANGELOG.md [hunk 12]
```diff
@@ -4,6 +4,51 @@ This changelog goes through all the changes that have been made in each release
...
+  * Change action GETs to POST (#10462) (#10464)
+* BUGFIXES
+  * Allow users with write permissions to modify issue descriptions and comments. (#10623) (#10626)
+  * Handle deleted base branch in PR (#10618) (#10619)
+  * Delete dependencies when deleting a repository (#10608) (#10616)
+  * Ensure executable bit is kept on the web editor (#10607) (#10614)
+  * Update mergebase in pr checker (#10586) (#10605)
+  * Fix release attachments being deleted while upgrading (#10572) (#10573)
+  * Fix redirection path if Slack webhook channel is invalid (#10566)
+  * head.tmpl og:image picture location (#10531) (#10556)
```
Comment (jolheiser) [inline CHANGELOG.md]: Just for consistency?
```suggestion
  * Fix head.tmpl og:image picture location (#10531) (#10556)
```

File: CHANGELOG.md (hunk 11 above)
Comment (zeripath) [reply CHANGELOG.md]: done

File: CHANGELOG.md [hunk 13]
```diff
@@ -4,6 +4,51 @@ This changelog goes through all the changes that have been made in each release
...
+  * Reading pull attachments should depend on read UnitTypePullRequests (#10346) (#10354)
+  * Set max-width on review-box comment box (#10348) (#10353)
+  * Prevent nil pointer in GetPullRequestCommitStatusState (#10342) (#10344)
+  * Fix protected branch status check settings (#10341) (#10343)
+  * Truncate long commit message header (#10301) (#10319)
+  * Set the initial commit status to Success otherwise it will always be Pending (#10317) (#10318)
+  * Don't manually replace whitespace during render (#10291) (#10315)
+* ENHANCEMENT
+  * Admin page for managing user e-mail activation (#10557) (#10579)
+  * update crypto vendors (#10385) (#10398)
```
Comment (jolheiser) [inline CHANGELOG.md]: Tiniest nit
```suggestion
  * Update crypto vendors (#10385) (#10398)
```

**Review by jolheiser — APPROVED** [review]
Tiny nits, otherwise LGTM

**Review by guillep2k — APPROVED** [review]

**Pull request merged**